internal/kafka/internal/workers/clusters_mgr_test.go

```diff
@@ -3884,3 +3927,213 @@ func TestClusterManager_reconcileClusterMachinePool(t *testing.T) {
 	}
 
 }
+
+func TestClusterManager_reconcileDynamicCapacityInfo(t *testing.T) {
+
+	type fields struct {
+		dataplaneClusterConfig *config.DataplaneClusterConfig
+		clusterService         services.ClusterService
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		arg     api.Cluster
+		wantErr bool
+	}{
+		{
+			name: "resets capacity to empty object when autoscalig is not enabled",
```

Comment:
Hey, there's just a tiny typo here: 
```diff 
- when autoscalig is not enabled
+ when autoscaling is not enabled
```
which is repeated for the following 4 test names also.

Other than that lgtm :) but I'd hold off for another reviewer

Reply:
Thanks @VanillaSpoon